frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -780,13 +780,26 @@ public static function apiList(\OmegaUp\Request $r) {
             ['promotion', 'demotion']
         );
 
-        return \OmegaUp\DAO\QualityNominations::getNominations(
+        $response = \OmegaUp\DAO\QualityNominations::getNominations(
             /* nominator */ null,
             /* assignee */ null,
             $offset,
             $rowCount,
             $types
         );
+
+        $pagerItems = \OmegaUp\Pager::paginate(
+            $response['totalRows'],
+            $rowCount,
+            $offset,
+            5,
+            []
+        );
+
+        return [
+            'nominations' => $response['nominations'],
+            'pagerItems' => $pagerItems,
```

Comment:
nit:

```suggestion
            'pager_items' => $pagerItems,
```

en las cosas que salen del API, se usa snake_case en vez de camelCase. igual en los otros lugares

Reply:
@lhchavez una vez ya me habias dicho que que usuara camelCase https://github.com/omegaup/omegaup/pull/3399#discussion_r373306619 pero en ese cambio es porque estaban dentro de payload o porque?

Reply:
correcto! en aquél caso era dentro del payload (donde sí se vale usar camelCase). en este caso es algo que se está regresnado del API, donde todo debería ser snake_case.

sorry si es un poco confuso u_u pero gracias por recordar el otro comentario!